### PR TITLE
changes as per issue 4036

### PR DESCRIPTION
**Pull request by guneetmutreja**
It contains required changes to be done to landcover mapping using sparse training data notebook as per discussion in issue #4036

### CODE REVIEW

File: samples/04_gis_analysts_data_scientists/land_cover_classification_using_sparse_training_data.ipynb [hunk 1]
```diff
@@ -91,13 +91,10 @@
   },
```
Comment (priyankatuteja) [inline samples/04_gis_analysts_data_scientists/land_cover_classification_using_sparse_training_data.ipynb:91]: <p>need to revert this back</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/840/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (priyankatuteja) [inline samples/04_gis_analysts_data_scientists/land_cover_classification_using_sparse_training_data.ipynb:91]: <p>You have removed the code for try it live that provides the training data. Can you please check this pull request again.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/840/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Review by priyankatuteja — CHANGES_REQUESTED** [review]
@guneetmutreja Please have a look at the changes made in the pr. Looks like when you resolved the conflicts, some unintentional things have gone wrong.

**Review by priyankatuteja — CHANGES_REQUESTED** [review]
@guneetmutreja Please have a look at the changes made in the pr. Looks like when you resolved the conflicts, some unintentional things have gone wrong.

**Comment by guneetmutreja** [comment]
Closing PR because some original changes were deleted. Will raise a new PR with the required changes.

**Pull request closed without merging**